lib/rubocop/config.rb

```diff
@@ -284,27 +284,45 @@ def pending_cops
       end
     end
 
+    # Returns target's locked gem versions (i.e. from Gemfile.lock or gems.locked)
+    # @returns [Hash{String => Gem::Version}] The locked gem versions, keyed by the gems' names.
+    def gem_versions_in_target
+      @gem_versions_in_target ||= read_gem_versions_from_target_lockfile
+    end
+
     def inspect # :nodoc:
       "#<#{self.class.name}:#{object_id} @loaded_path=#{loaded_path}>"
     end
 
     private
 
     def target_rails_version_from_bundler_lock_file
-      @target_rails_version_from_bundler_lock_file ||= read_rails_version_from_bundler_lock_file
```

Comment:
I removed this memoization because:

1. If `read_rails_version_from_bundler_lock_file` returned `nil`, it wouldn't memoize correctly and would be trying to parse the lockfile on every call anyway
2. This value is only accessed from `#target_rails_version`, which is itself memoized